### PR TITLE
Badge component

### PR DESCRIPTION
**Pull request by aamir74**
added avatar and icon badges

### CODE REVIEW

**Review by ruchiragoswami — APPROVED** [review]
Looks good Aamir :)

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  border: 1px solid var(--gray-border);
+  padding: 2rem;
+  border-radius: 10px;
+  text-align: center;
+  margin: var(--margin-width);
+  display: flex;
+}
+.avatar {
+  display: flex;
+  flex-direction: row;
```
Comment (kart-c) [inline components/badge.css:11]: You can remove this line of code as `row` is the default value for `flex-direction`

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+.button_main {
+  border: 1px solid var(--gray-border);
+  border-radius: 5px;
+  margin: var(--margin-width);
+  padding: 1rem;
+  display: flex;
+}
+.action-button {
+  display: flex;
+  flex-direction: row;
```
Comment (kart-c) [inline components/badge.css:44]: You can remove this one too

**Review by kart-c — APPROVED** [review]
Looks good. You can merge it

**Pull request merged**